add_me_to_your_PATH/sysimage.jl

```diff
@@ -16,7 +16,7 @@ end
 # Skip generating a system image when there are no dependencies
 isempty(project.dependencies) && exit(0)
 
-Pkg.add("PackageCompiler")
+Pkg.add(name="PackageCompiler", version="1.7.7")
```

Comment:
We want to get any future version 1 fixes:
```suggestion
Pkg.add(name="PackageCompiler", version="1")
```
According to the `Pkg.add` docstring specifying the entire version number selects an exact version